android/sample/build.gradle

```diff
@@ -13,11 +13,17 @@ android {
     lintOptions {
         abortOnError false
     }
+
     sourceSets {
         main {
             manifest.srcFile './AndroidManifest.xml'
         }
     }
+
+    compileOptions {
+        targetCompatibility = "8"
```

Comment:
@passy You can use `JavaVersion.VERSION_1_8` as an alternative for this kind of Java version constants :)

Reply:
Oh, good idea. Thanks!